homeassistant/components/miele/strings.json

```
@@ -1088,6 +1088,9 @@
     "invalid_target": {
       "message": "Invalid device targeted."
     },
+    "oauth2_implementation_unavailable": {
+      "message": "OAuth2 implementation temporarily unavailable, will retry"
```

Perhaps we could remove "temporarily". Actually, we don't know that until the retries succeed or fail.
```suggestion
      "message": "OAuth2 implementation unavailable, will retry"
```